### ISSUE TITLE
Inconsistent requirements files

### ISSUE THREAD

**Issue by gantir** [OpenMined/PyGrid#416]
**Describe the bug**
There are 3 requirments.txt files 

- . (root)
  - requirements.txt and dev-requirements.txt
- app/web
- gateway

It is not clear as to which packages are needed where. Either have common requirments.txt at root level and individual requirments.txt internally or have different. The current approach is not best and I believe is causing issues when one is trying to install from code following the instructions.
 
**To Reproduce**
1. Go to root folder. 
2. Follow the steps to docker build the node.
3. Once the step 6 is underway, it throws the error
4.
```
ERROR: Could not find a version that satisfies the requirement pysyft_proto (from grid==0.1.0.0a1.dev422->-r requirements.txt (line 7)) (from versions: none)
ERROR: No matching distribution found for pysyft_proto (from grid==0.1.0.0a1.dev422->-r requirements.txt (line 7))
on my local pip install . also gave same error, but install pysft and proto manually removed it.
```

**Expected behavior**
No error and successful build of node

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [macOS Mojave]
 - Version [10.14.6]

**Additional context**

**Comment by IonesioJunior** [comment]
Yeah! You're correct @gantir!
Nowadays, we're developing three different components in the same repository (PyGrid Library, Grid Node, Grid Gateway).
It's a little bit confusing to handle three different dependencies at the same repository. So, in the near future, we'll split it into different repositories. 
- **[WIP]** We intend to move our PyGrid lib to PySyft as a new module, you can follow the progress [here](https://github.com/OpenMined/PySyft/pull/2760) 
- **[TODO]** Create a new repository to Grid Nodes.
- **[TODO]** Keep this repository to develop our Grid Gateway. It'll connect every type of syft workers (Grid Nodes, syft.js, and AndroidWorkers)

**Comment by gantir** [comment]
@IonesioJunior ,

That makes sense. If you can create a new repo for Grid Nodes, I can take up moving the code there. For now it will have both PySyft and PyGrid as dependencies and later it can be moved to PySyft alone.

Once PyGrid move to PySyft is merged, this repo will be left with Grid Gateway alone.
